### PR TITLE
Feature/mulit choice slash commands

### PR DESCRIPTION
**Pull request by StarBlasto**
Added roulette and test command making use of options and multiple choice options

### CODE REVIEW

File: Commands/Gambling/roulette.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# /commands/basic/help.py
+import discord
+from discord.ext import commands
+from typing import Literal
+import random as r
```
Comment (Cosmo517) [inline Commands/Gambling/roulette.py]: Random shouldn't be left as just `r`, instead name it something more readable, like `rand`

File: Commands/Gambling/roulette.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    Parameters: None
+    Returns: None
+    """
+    # variables
+    red_numbers = {1, 3, 5, 7, 9, 12, 14, 16, 18, 19, 21, 23, 25, 27, 30, 32, 34, 36}
+    black_numbers = {2, 4, 6, 8, 10, 11, 13, 15, 17, 20, 22, 24, 26, 28, 29, 31, 33, 35}
+    green_numbers = {0, 37}
+    rand_num = r.randint(0, 37)
+    col = discord.Color.blurple()
```
Comment (Cosmo517) [inline Commands/Gambling/roulette.py]: Should probably name this as color, or something similar. `col` to me makes me think column at first. Not a huge deal though.

Comment (StarBlasto) [reply Commands/Gambling/roulette.py]: updated

File: Commands/Gambling/roulette.py (hunk 1 above)
Comment (StarBlasto) [reply Commands/Gambling/roulette.py]: updated

**Pull request merged**